crates/pipeline_manager/src/api/program.rs

```diff
@@ -52,11 +43,11 @@ pub(crate) struct CompileProgramRequest {
     version: Version,
 }
 
-/// Request to create a new Feldera program.
+/// Request to create a new program.
 #[derive(Debug, Deserialize, ToSchema)]
 pub(crate) struct NewProgramRequest {
     /// Program name.
-    #[schema(example = "Example program")]
+    #[schema(example = "example-program")]
```

Comment:
Is this implying that spaces are not allowed?
Are the constraints on this string?

Reply:
There are currently no constraints on naming, though names are now used directly in the URL API paths